src/data/nat/log.lean

```diff
@@ -150,13 +148,10 @@ end
 
 lemma log_le_log_of_left_ge {b c n : ℕ} (hc : 1 < c) (hb : c ≤ b) : log b n ≤ log c n :=
 begin
-  cases n, { simp },
-  rw ← pow_le_iff_le_log hc (zero_lt_succ n),
-  calc
-    c ^ log b n.succ ≤ b ^ log b n.succ : pow_le_pow_of_le_left
-                                            (le_of_lt $ zero_lt_one.trans hc) hb _
-                 ... ≤ n.succ           : pow_log_le_self (lt_of_lt_of_le hc hb)
-                                            (zero_lt_succ n)
+  cases n, { rw [log_zero_right, log_zero_right] },
+  rw ←pow_le_iff_le_log hc (zero_lt_succ n),
+  exact (pow_le_pow_of_le_left (zero_lt_one.trans hc).le hb _).trans
+    (pow_log_le_self (hc.trans_le hb) (zero_lt_succ n))
```

Comment:
This part of the proof doesn't seem like an improvement to me. It's shorter, but the previous proof is more readable.